<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Курс валют</string>
  </property>
  <widget class="QGroupBox" name="eth_dollar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Etherium to Dollars</string>
   </property>
   <widget class="QLabel" name="eth_dollars_course">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>26</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="eth_dol_value">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>22</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="eth_rub">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Etherium to Rub</string>
   </property>
   <widget class="QLabel" name="eth_rub_course">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>25</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="eth_rub_value">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>24</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="bit_dollar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Bitcoin to Dollars</string>
   </property>
   <widget class="QLabel" name="bicoin_dollars_course">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>25</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="bitcoin_dol_value">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>22</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="bit_rub">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Bitcoin to Rub</string>
   </property>
   <widget class="QLabel" name="bicoin_rub_course">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>25</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="bitcoin_rub_value">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>22</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="refresh">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
